apps/ui/src/components/AuctionUserBid.vue

```diff
@@ -0,0 +1,108 @@
+<script setup lang="ts">
+import { formatPrice, Order } from '@/helpers/auction';
+import { AuctionDetailFragment } from '@/helpers/auction/gql/graphql';
+import { _c, _n, _p, _t } from '@/helpers/utils';
+
+const props = defineProps<{
+  auctionId: string;
+  auction: AuctionDetailFragment;
+  order: Order;
+  orderStatus?: 'open' | 'filled' | 'partially-filled' | 'rejected' | 'claimed';
+  biddingTokenPrice: number;
+}>();
+
+const emit = defineEmits<{
+  (e: 'cancel', order: Order): void;
+}>();
+
+const isCancellable = computed(() => {
+  return Number(props.auction.orderCancellationEndDate) * 1000 > Date.now();
+});
+
+const orderPercentage = computed(() => {
+  return (
+    Number(props.order.buyAmount) / Number(props.auction.exactOrder.sellAmount)
+  );
+});
+</script>
+
+<template>
+  <div
+    class="flex justify-between items-center gap-3 py-3 px-4 relative leading-[22px]"
+  >
+    <div
+      class="right-0 top-0 h-[8px] absolute choice-bg opacity-20 _1"
+      :style="{
+        width: `${Math.min(orderPercentage * 100, 100).toFixed(2)}%`
+      }"
+    />
+    <div class="min-w-[100px] text-skin-link truncate">
+      <span v-if="orderStatus === 'open'">Open</span>
+      <span v-else-if="orderStatus === 'claimed'" class="text-skin-success">
+        Claimed
+      </span>
+      <span v-else-if="orderStatus === 'filled'" class="text-skin-success">
+        Filled
+      </span>
+      <span
+        v-else-if="orderStatus === 'partially-filled'"
+        class="text-skin-success opacity-70"
+      >
+        Partially-filled
+      </span>
```

Comment:
I am bit color blind but i feel partially filled look more greenish than filled 😅 

<img width="767" height="368" alt="Image" src="https://github.com/user-attachments/assets/01259105-6b0d-46ef-aafa-9105c41c181b" />

maybe a neutral color `text-yellow-500` ?

<img width="774" height="371" alt="Image" src="https://github.com/user-attachments/assets/653ca17f-4c10-441f-bd40-1fd4ea528292" />


NOT A DESIGNER, CAN IGNORE

Reply:
I think it looks better (haven't tested on light mode). Will update it according to your suggestion, this [might change ](https://github.com/snapshot-labs/sx-monorepo/issues/1778)in the future as well.